### ISSUE TITLE
Acute posterior multifocal placoid pigment epitheliopathy 

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6075]
**Preferred term label:**
Acute posterior multifocal placoid pigment epitheliopathy 

**Synonyms**
APMPPE

**Definition (free text, please give PubMed ID)**
Acute posterior multifocal placoid pigment epitheliopathy (APMPPE) is a rare condition affecting young adults around 20-30 years of age. It has a self-limiting course, usually involves both eyes unequally, and the inflammation is involving choriocapillaris, the retinal pigment epithelium (RPE) and the outer retina.
PMID: 27373117 

**Parent term (use hpo.jax.org/app)**
Abnormal chorioretinal morphology

**Comment by pnrobinson** [comment]
This is a disease that has many different phenotypic features. Please use the MONDO term for the disease (https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0043089). I believe the HPO already has all terms needed to describe the phenotypic features that characterize this disease.
